gumbo-parser/fuzzer/build.sh

```diff
@@ -0,0 +1,46 @@
+export SANITIZER_OPTS=""
+export SANITIZER_LINK=""
+
+if [ -z "${LLVM_CONFIG}" ]
+then
+  echo '$LLVM_CONFIG has not been configured, expecting "export LLVM_CONFIG=/usr/bin/llvm-config-12" assuming clang-12 is installed, however any clang version works'
+  exit
+fi
```

Comment:
My (debian-based) system has a bare `llvm-config` which is a symlink to the active installation (llvm-14). Can we simply use that if it's present?

Reply:
fixed and tested